packages/core/src/VNodes/VNode.ts

```diff
@@ -10,9 +10,9 @@ export enum RelativePosition {
 }
 
 export enum VNodeType {
-    NODE = 'node',
-    MARKER = 'marker',
-    FRAGMENT = 'fragment',
+    NODE = 'Node',
```

Comment:
:eyes:

Reply:
![image](https://user-images.githubusercontent.com/24205914/72426707-8c795d80-378a-11ea-99e1-65f15f7e7dfa.png)

Reply:
Now that I realized (thanks to the list PR) that you are defining these to display them in the DevTools, I might revert this change such that the devtools looks a bit more like the "css" part of Chrome's.